src/briefcase/commands/dev.py

```diff
@@ -192,6 +201,21 @@ def get_environment(self, app: AppConfig):
 
         return env
 
+    def virtual_environment(self, appname: str, isolated: bool) -> virtual_environment:
+        """Create and return a virtual environment context for the app.
+
+        :param appname: The name of the app to create a venv context for
+        :param isolated: Whether to create an isolated virtual environment
+        :returns: A virtual environment context manager either VenvEnvironment or
+            NoOpEnvironment based on isolation setting
+        """
+        return virtual_environment(
+            tools=self.tools,
+            console=self.console,
+            venv_path=self.base_path / ".briefcase" / appname / "venv",
```

Comment:
Flagging that the web dev backend will likely want to use a different venv name, because it will have different set of requirements.